inference-engine/tests/unit/gna/ngraph/transformations/gna_split_convolution_with_large_buffer_size.cpp

```diff
@@ -0,0 +1,597 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "transformations/split_convolution_with_large_buffer_size.hpp"
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include <ngraph/function.hpp>
+#include <ngraph/opsets/opset7.hpp>
+#include <ngraph/pass/manager.hpp>
+#include <transformations/init_node_info.hpp>
+
+namespace testing {
+
+// use constexpr uint32_t bufferMaxSize = 65528 from gna_limitations
+
+TEST(TransformationTests, SplitConvolutionTest) {
+    std::shared_ptr<ngraph::Function> func(nullptr), reference_func(nullptr);
+
+    {
+        auto input_params = std::make_shared<ngraph::opset7::Parameter>(ngraph::element::f32,
```

Comment:
is it possible here to create some generic function which creates a subgraph with 1 convolution or with several convolutions and just gets shapes and a number of convolutions?